pipeline/schemas/environment.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime
+from typing import List, Optional
+
+from pipeline.schemas.base import BaseModel
+
+
+class EnvironmentCreate(BaseModel):
+    name: str
+    python_requirements: List[str]
+
+
+class EnvironmentBrief(BaseModel):
+    id: str
+    name: str
+    deleted_at: Optional[datetime]
+
+
+class EnvironmentGet(BaseModel):
+    id: str
+    name: str
+    python_requirements: List[str]
+    locked: bool
```

Comment:
These could be updated; e.g. the API also returns `created_at` and `updated_at`.

Reply:
👍